README.md

```diff
@@ -25,6 +25,7 @@ conditions. See the [LICENSE] file for details.
 ## Prerequisites
 
 1. Visual Studio Code with "Remote Development" installed
+2. [Set-up Git credential sharing with your container](GIT-SETUP.md)
```

Comment:
Use this phrasing: Shareable git credentials with the devcontainer

Reply:
Just curious, why this phrasing?